backend/data_distro/api_tests/schema/Makefile

```diff
@@ -0,0 +1,6 @@
+all:
```

Comment:
Once we move everything into the centralized schema directory, we rely on [that makefile](https://github.com/GSA-TTS/FAC/blob/main/backend/schemas/Makefile) instead. This one isn't needed for anything anyway.